app/config.js

```diff
@@ -1,36 +1,45 @@
-// This is the default configuration for the dev mode of the web console.
-// A generated version of this config is created at run-time when running
-// the web console from the openshift binary.
-//
-// To change configuration for local development, copy this file to
-// assets/app/config.local.js and edit the copy.
-window.OPENSHIFT_CONFIG = {
-  apis: {
-    hostPort: "localhost:8443",
-    prefix: "/apis"
-  },
-  api: {
-    openshift: {
-      hostPort: "localhost:8443",
-      prefix: "/oapi"
+'use strict';
+
+(function() {
+  // This is the default configuration for the dev mode of the web console.
+  // A generated version of this config is created at run-time when running
+  // the web console from the openshift binary.
+  //
+  // To change configuration for local development, copy this file to
+  // assets/app/config.local.js and edit the copy.
+  var hostIP = 'localhost';
+  var hostPort = ':8443';
```

Comment:
Just the next line, but setting it up for anything we might do in the future. I can collapse if we don't think so.